src/rtc.c

```diff
@@ -203,6 +205,20 @@ static void RTC_initClock(sourceClock_t source)
   __HAL_RCC_RTC_ENABLE();
 }
 
+#if defined(STM32F1xx)
+/**
+  * @brief set user (a)synchronous prescaler values.
+  * @note  use RTC_AUTO_1_SECOND to reset value and use computed ones
+  * @param asynch: asynchronous prescaler value in range 0 - PREDIVA_MAX
```

Comment:
`PREDIVA_MAX` is not defined for STM32F1. It could be fine to define it for F1 with `0xFFFFFU`

https://github.com/stm32duino/STM32RTC/blob/38ce89132f9be35ad7411ffbcefbd31cb2a5c3e4/src/rtc.h#L89-L97